framework/LoopMeUnitedSDK/AdConfiguration/AdConfiguration/VAST/AdTrackingLinks.swift

```diff
@@ -8,43 +8,63 @@
 
 import Foundation
 
-struct AdTrackingLinks {
-    
-    static func +=(lhs: inout AdTrackingLinks, rhs: AdTrackingLinks) {
-        lhs.errorTemplates.formUnion(rhs.errorTemplates)
-        lhs.verificationNotExecuted.formUnion(rhs.verificationNotExecuted)
-        lhs.impression.formUnion(rhs.impression)
-        lhs.creativeViewCompanion.formUnion(rhs.creativeViewCompanion)
-        lhs.clickVideo = rhs.clickVideo
-        lhs.clickCompanion = rhs.clickCompanion
-        lhs.linear += rhs.linear
+public struct ProgressEvent: Hashable {
+
+    public static func == (lhs: ProgressEvent, rhs: ProgressEvent) -> Bool {
+        return lhs.link == rhs.link && lhs.offset == rhs.offset
```

Comment:
We can improve this to 
`public static func == (lhs: LinearTracking, rhs: LinearTracking) -> Bool {
    return lhs.allFields() == rhs.allFields()
}`